### DISCUSSION TITLE
Swipe up to refresh an sms thread

### DISCUSSION THREAD

**Discussion by getsean** [0perationPrivacy/VoIPSuite-Mobile#36] [Ideas]
As of now when I'm in an sms thread it does not show me incoming sms in real time. You have to go to main screen for them to populate. I know this might be fixed but what about a "swipe up to refresh" while in the thread to get new sms to how up faster?

**Comment by 0perationPrivacy** [comment]
sockets have been implemented to update it realtime, so a refresh wont be needed. We're troubleshooted it and fixed it in the code. It will be fixed in the next release.
